Update Community Cloud docs (#1149)
streamlit/docs: content/deploy/community-cloud/manage-your-account/sign-in-sign-out.md

# Sign in & sign out

> Once you've created your account, you can sign in to <a href="https://share.streamlit.io" target="_blank">share.streamlit.io</a> as described by the options below.

Reviewer: ```suggestion
After you've created your account, you can sign in to <a href="https://share.streamlit.io" target="_blank">share.streamlit.io</a> as described by the following options.
```

![Sign in to Streamlit Community Cloud](/images/streamlit-community-cloud/account-sign-in.png)

## Sign in with Google

1. Click "**Continue to sign-in**."

> 1. Click "**Continue with Google**".

Reviewer: ```suggestion
1. Click "**Continue with Google**."
```

1. Enter your Google account credentials and follow the prompts.

If your account is already linked to GitHub, you may be immediately prompted to sign in with GitHub.

## Sign in with GitHub

1. Click "**Continue to sign-in**."

> 1. Click "**Continue with GitHub**".

Reviewer: ```suggestion
1. Click "**Continue with GitHub**."
```
